pkg/m3admin/placement/placement.go

```diff
@@ -39,51 +37,18 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	placementBaseURL   = "/api/v1/services/m3db/placement"
```

Comment:
Is there a `const` that can used from the `m3` repository for this ?

Reply:
Yea this can break with either approach. I guess this is when pinning the version of M3 is going to be very important for either approach.